### ISSUE TITLE
Exhuman, Swarmanoid Issues

### ISSUE THREAD

**Issue by Divergent0** [rbewley4/ep-character-creator#75]
Exhuman currently doesn't allow you to choose the combat skill to give +20 to, or the exotic field to give that +20 bonus to.

Certain Swarmanoid augments are missing, such as Jaws, or listed under Gear as Misc.

**Comment by EmperorArthur** [comment]
Hey there,

Could you give me a list of what exactly is missing, or misplaced.  I can update the database, but don't know exactly what's broken.

A spreadsheet or csv file would be ideal.

**Comment by Jundiii** [comment]
Hi. I confirm Divergent0 bug.
If you select the Faction EXHUMAN, it should grant you:

- [x] 1. +10 FRAY Skill
- [x] 2. +10 PSYCHOSURGERY
- [ ] 3. +20 on one Exotic [field] Skill
- [ ] 4. +20 on one combat [field] skill
- [x] 5. +10 MEDICINE [field]

The 3rd and 4th bonus are missing.

Moreover the EXHUMAN should have some flow (such as modified behaviour) that is not mentioned in the description nor in the faction stats.

**Comment by EmperorArthur** [comment]
Hmm, looking at it the database has everything, but it won't let you pick any of the skills aside from medicine.

Also, it does add the flaws, but doesn't mention it in the description.

**Comment by EmperorArthur** [comment]
The version at https://www.cd-net.net/ep-character-creator/src/Creator/version4/index.php has fixed the traits not displaying issue, but the bonus selection still needs work.
